File: poo_java/Persona.java
class Persona {
    //atributos
   protected Integer idpersona;
   protected String nombre;
   protected String paterno;
   protected String materno;
    //constructor
    public Persona(String nombre){
      this.nombre = nombre;
  }
    // metodos
    public String getNombre() {
      return this.nombre;
    }
    public Integer getIdpersona() {
      return this.idpersona;
    }
    public void setIdpersona(Integer idpersona) {
      this.idpersona = idpersona;
    }
    public String getPaterno() {
      return this.paterno;
    }
  
    public void setPaterno(String paterno) {
      this.paterno = paterno;
    }
    public String getMaterno() {
      return this.materno;
  }
  public void setMaterno(String materno) {
    this.materno = materno;
  }
  }

File: poo_csharp/Persona.cs

namespace poo_csharp
{
    public class Persona
    {
        protected int idpersona;
        public string nombre;
        protected string paterno;
        protected string materno;
        
        

        // constructor
        public Persona(string nombre){
         this.nombre = nombre;
        }
        public string getNombre(){
            return this.nombre;
        }
           public string getPaterno(){
            return this.paterno;
        }
        public string setPaterno(string paterno){
            return this.paterno =paterno;
        }
    }
}